vars/CentOS_7.yml

```diff
@@ -6,6 +6,7 @@ blivet_package_list:
   - libblockdev-lvm
   - libblockdev-mdraid
   - libblockdev-swap
+  - vdo
```

Comment:
I don't think LVM VDO support was backported to CentOS/RHEL 7 and we don't support standalone VDO devices in blivet so this should probably be removed.

Reply:
Fixed: `vdo` removed from `CentOS_7.yml` and `RedHat_7.yml`.